src/Examples/Advanced/CustomProperties/Hydrocarbon_Processing_example.ipynb

```diff
@@ -151,6 +151,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
+      "2021-05-14 15:17:11 [INFO] idaes.generic_models.properties.core.generic.generic_property: DEPRECATED - definiton of generic property packages is moving to using static classes instead of modules. Please refer to the IDAES documentation.\n",
```

Comment:
As before - lets try to fix the underlying issue (if they use the same property package, fixing one will fix both).